dev-db/postgresql/postgresql-11.20.ebuild

```diff
@@ -386,7 +386,14 @@ pkg_config() {
 	einfo "Initializing the database ..."
 
 	if [[ ${EUID} == 0 ]] ; then
-		su - postgres -c "${EROOT}/usr/$(get_libdir)/postgresql-${SLOT}/bin/initdb -D \"${DATA_DIR}\" ${PG_INITDB_OPTS}"
+		local cmd="${EROOT}/usr/$(get_libdir)/postgresql-${SLOT}/bin/initdb -D \"${DATA_DIR}\" ${PG_INITDB_OPTS}"
+		if [[ -x /usr/bin/su ]]; then
```

Comment:
This applies to all files.

Rather than testing whether an absolute path is executable, use `command`. A false negative may be returned if  `su` isn't in the expected place. (`su`'s absolute path on my system is `/bin/su`.)

Instead, use `command` like so: `if command -v su >/dev/null 2>&1 ; then ...`